src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyCacheMaxSizeAction.java

```diff
@@ -76,9 +91,9 @@ public long coolOffPeriodInMillis() {
     @Override
     public Map<NodeKey, ImpactVector> impact() {
         final ImpactVector impactVector = new ImpactVector();
-        if (desiredCapacityInBytes > currentCapacityInBytes) {
+        if (desiredCacheMaxSizeInBytes > currentCacheMaxSizeInBytes) {
```

Comment:
Two things we also need to account before increasing/decreasing the cache size :
1. Field Data Cache Circuit Breaker -  The default value for field data circuit breaker is configured at 40% of the maximum JVM heap and is managed via static setting `indices.breaker.fielddata.limit`. If the circuit-breaker limit is lower than the cache size, no data will ever be evicted. In order for it to work properly, the circuit breaker limit must be higher than the cache size and we should assert for it when increasing the cache size. 
2. Explicit Cache size set by end user. - In case customer sets the explicit static value for cache size, how do we account for them in our actions ?

Reply:
Issue to track this:
https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/issues/320